packages/web-components/src/select/select.ts

```diff
@@ -0,0 +1,206 @@
+import { attr, FASTElement, observable } from '@microsoft/fast-element';
+import { ARIAGlobalStatesAndProperties } from '@microsoft/fast-foundation';
+import { SelectSize } from './select.options.js';
+
+/**
+ * The base class used for constructing a fluent-slider custom element
+ * @public
+ */
+export class Select extends FASTElement {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleInput = this.handleInput.bind(this);
+    Object.assign(this, new ARIAGlobalStatesAndProperties()); // Assigns ARIA global states and properties to the element
```

Comment:
This should be done outside the class.

Reply:
I'm still a little lost on this. I copied and modified [the code from the FAST Foundation TextField component](https://github.com/microsoft/fast/blob/864303b923a10fd8b1d88444efb05e3f64c9524d/packages/web-components/fast-foundation/src/text-field/text-field.ts#L256-L280). Let me know if this does the trick. If not, maybe we can pair or chat to explain how to apply this mixin?

Reply:
The `applyMixins` which is commented out is where this would be done as opposed to in the constructor.

Reply:
I can also look to create a PR against your branch if need be.

Reply:
I uncommented it and changed it to match the TextField implementation.

Reply:
Similar to the FormAssociated question: What can I do on my end in the browser to test if this is working correctly? I'd like to get better at debugging, but I'm not sure what I need to test for to ensure I have this implemented correctly.